deploy/ocm-agent-operator-managednotifications/node-condition/200-sre-node-condition-managed-notification.ManagedNotification.yaml

```
@@ -11,31 +11,31 @@ spec:
       severity: Warning
       summary: "Worker node is experiencing DiskPressure"
       activeBody: |-
-        DiskPressure condition is true for at least one worker.
+        DiskPressure condition is true for at least one worker. Consider moving application storage off the worker node and into persistent volumes. For more information, see [Configuring persistent storage](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/storage/configuring-persistent-storage).
       resolvedBody: |-
         DiskPressure condition is resolved.
       resendWait: 6
     - name: NodeConditionMemoryPressureNotification
       severity: Warning
       summary: "Worker node is experiencing MemoryPressure"
       activeBody: |-
-        MemoryPressure condition is true for at least one worker.
+        MemoryPressure condition is true for at least one worker. For more information on memory management in OpenShift applications, see [Understanding managing application memory](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/cluster_administration/nodes#nodes-cluster-resource-configure-about_nodes-cluster-resource-configure).
       resolvedBody: |-
         MemoryPressure condition is resolved.
       resendWait: 6
     - name: NodeConditionPIDPressureNotification
       severity: Warning
       summary: "Worker node is experiencing PIDPressure"
       activeBody: |-
-        PIDPressure condition is true for at least one worker.
+        PIDPressure condition is true for at least one worker. This means that more than the maximum allowed number of processes are present on a worker node. This is likely to create instability in the node, as required system services may not be able to create new processes. Reduce the number of pods on the node, or the number of processes generated by applications on the node.
```

```suggestion
        PIDPressure condition is true for at least one worker. This means that more than the maximum allowed number of process identifiers are consumed on one or more worker nodes. This is likely to create instability in the node, as required system services may not be able to create new processes. Reduce the number of pods scheduled to the impacted nodes, or reduce the number of process identifiers consumed by applications on the impacted nodes.
```

No KCS on what PIDs are and how they're consumed?